README.md

```diff
@@ -26,37 +26,32 @@ The named arguments are captured as key/value properties that can be rendered in
 
 ```json
 {
-  "Timestamp": "2016-03-17T00:17:01Z",
-  "Level": "Information",
-  "MessageTemplate": "Hello, {name}!",
-  "Properties": {
-    "name": "nblumhardt",
-    "target": "web_we_are"
-  }
+  "@t": "2016-03-17T00:17:01Z",
+  "@mt": "Hello, {name}!",
+  "name": "nblumhardt",
+  "target": "web_we_are"
 }
 ```
 
 This makes log searches in an appropriate back-end collector much simpler:
 
 ![Event in Seq](https://raw.githubusercontent.com/nblumhardt/emit/master/asset/event_in_seq.png)
 
-[Seq](https://getseq.net) and its JSON format are implemented for the work-in-progress here, but support for other log collectors and formats is on its way.
-
-If you don't have Seq running, events can be written to `io::stdout` instead:
+Events can be written to `io::stdout` in a number of formats:
 
 ```rust
 use emit::collectors::stdio::StdioCollector;
+use formatters::text::PlainTextFormatter;
+
 let _flush = PipelineBuilder::new()
-    .write_to(StdioCollector::new())
+    .write_to(StdioCollector::new(PlainTextFormatter::new()))
```

Comment:
Since `PlainTextFormatter` is a zero sized type I think you could call `StdioCollector::new (PlainTextFormatter)`. But for docs it might be nice to keep a consistent API for formatters that do have a size and need a constructor. What do you think?